### ISSUE TITLE
v4.0.2 Release is broken

### ISSUE THREAD

**Issue by ProTip** [yargs/y18n#124]
Index is not built:
```
/node_modules/y18n/build$ ls
index.cjs  lib
```

**Comment by bcoe** [comment]
@ProTip working fine for me:

```
  },
  "type": "module",
  "version": "4.0.2"
}
bencoe-macbookpro3:install bencoe$ node
Welcome to Node.js v12.21.0.
Type ".help" for more information.
> const y18n = require('y18n')
undefined
> y18n()
{
  __: [Function: bound __],
  __n: [Function: bound __n],
  setLocale: [Function: bound setLocale],
  getLocale: [Function: bound getLocale],
  updateLocale: [Function: bound updateLocale],
  locale: 'en'
}
```

☝️ can you provide an example of a project that is failing?

**Comment by bcoe** [comment]
fixed by `4.0.3`:

```
npm notice 
npm notice 📦  y18n@4.0.3
npm notice === Tarball Contents === 
npm notice 731B  LICENSE     
npm notice 5.3kB index.js    
npm notice 889B  package.json
npm notice 1.3kB CHANGELOG.md
npm notice 2.8kB README.md   
npm notice === Tarball Details === 
npm notice name:          y18n                                    
npm notice version:       4.0.3                                   
npm notice package size:  4.3 kB                                  
npm notice unpacked size: 11.0 kB                                 
npm notice shasum:        b5f259c82cd6e336921efd7bfd8bf560de9eeedf
npm notice integrity:     sha512-JKhqTOwSrqNA1[...]Fmlb/U4UZ4hJQ==
npm notice total files:   5                                       
npm notice 
+ y18n@4.0.3
bencoe-macboo
```
